server/api.py

```diff
@@ -141,7 +141,7 @@ class Vcon(BaseModel):
 async def get_vcons():
     r = redis_mgr.get_client()
     keys = await r.keys("vcon:*")
-    uuids = [key.replace("vcon:", "") for key in keys]
+    uuids = [key.decode().replace("vcon:", "") for key in keys]
```

Comment:
I do not understand why the uuid is UTF-8 encoded.  The keys should be strings. Hence the substring should still be a string.  This sounds like a bug where something has incorrectly UTF-8 encoded or converted the UUID into binary that should not have.